portfolio/src/main/webapp/index.html

```diff
@@ -2,18 +2,42 @@
 <html>
```

Comment:
nit: Your commit says you changed the readme, but this isn't the readme. Not sure how you can change a commit message but google should definitely be able to tell you.

Reply:
I was just messing around with the commits trying to understand them more and this was an error on my part